pkgs/development/python-modules/subunit/default.nix

```diff
@@ -24,7 +24,7 @@ buildPythonPackage {
   checkInputs = [ testscenarios hypothesis fixtures pytest ];
 
   # requires unittest2, which no longer supported in 3.10
-  doCheck = pythonOlder "3.10";
+  doCheck = pythonAtLeast "3.10";
```

Comment:
This change looks odd.

Reply:
I accidently pushed, when I meant to do a build of the package. But yea, I should be doing stabilization while tired....